src/features/user/UserProfile/components/MyTrees/MyTrees.tsx

```diff
@@ -43,7 +47,11 @@ export default function MyTrees({ profile }: Props): ReactElement {
       Array.isArray(contributions) &&
       contributions.length !== 0 ? (
         <div className={styles.myTreesSection}>
-          <div className={styles.myTreesTitle}>{t('me:myForest')}</div>
+          <div className={styles.myTreesTitle}>
+            {authenticatedType === 'private'
+              ? t('me:myForest')
+              : t('me:nameForest', { name: profile.displayName })}
```

Comment:
If I login as a TPO is does not say `My Forest`, but `<Name>'s Forest` looking at the own profile. Is that ok?

Reply:
Fixed it with https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/562/commits/4c1b9500a6913bfcfecc85b87da40ca3a0878e6c

Reply:
No, but I can see you fixed it :)